lib/rbac/filterer.rb

```diff
@@ -417,17 +402,33 @@ def scope_to_tenant(scope, user, miq_group)
       tenant_id_clause ? scope.where(tenant_id_clause) : scope
     end
 
+    ##
+    # Main scoping method
+    #
     def scope_targets(klass, scope, rbac_filters, user, miq_group)
+      # Results are scoped by tenant if the TenancyMixin is included in the class,
+      # with a few manual exceptions (User, Tenant). Note that the classes in
+      # TENANT_ACCESS_STRATEGY are a consolidated list of them.
       if klass.respond_to?(:scope_by_tenant?) && klass.scope_by_tenant?
```

Comment:
> Vm.with_security(:user => X)

Don't know how I feel about that, specifically, but...

> I'd prefer to use rbac lookup tables over methods.

For sure. Personally I'd like to see a hook you can have on classes to register with RBAC, so it has a lookup table that classes obviously opt-in to and not keep a hardcoded list.

But that's beyond the conversation here :)  @kbrock and @Fryguy Let me know what/if you'd like to change about this comment, but not changing the implementation from what it was (this), for now. 💙